source/_integrations/husqvarna_automower.markdown

```diff
@@ -107,7 +107,7 @@ The integration will create the following binary sensors:
 
 ### Button (if available)
 
-The integration will create a button entity for confirming minor mower errors. This entity is disabled by default. You have to enable it manually. The API can't detect if the mower has the capability to confirm minor errors remotely. Before enabling this function, refer to the mower documentation.
+The integration will create a button entity for confirming minor mower errors.
```

Comment:
Should we clarify here that the button only exist if supported by the mower?

Reply:
@gjohansson-ST, are you good with the note in the title?

Reply:
Didn't see that. I'm good 👍